### PR TITLE
gui/install dialog: Optimize text again

### PR DESCRIPTION
**Pull request by Croden1999**
- gui/install dialog: Optimize text again
- lang: update other lang of vita3k_update
based on psvita lang and used psvita check again

Optimized #2432 , should be display failed install and check, please Mr.@Zangetsu38 check this code

image

### CODE REVIEW

**Comment by Croden1999** [comment]
> I no understund ? Is totaly wrong here



> I no understund ? Is totaly wrong here

Mr. Zangetsu, idk how to explain, just want to more clearly

**Comment by Zangetsu38** [comment]
> > I no understund ? Is totaly wrong here
> 
> Mr. Zangetsu, idk how to explain, just want to more clearly in install_failed i found a bug in license install dialog, if select wrong license file or enter zRIF key, Vita3K will be crash.

i know, is old, i have no idea how fix it lol

**Comment by CoffeeBrewer64** [comment]
what should I do about my pr then?

**Comment by Croden1999** [comment]
> what should I do about my pr then?

keep it, i not change other

**Comment by CoffeeBrewer64** [comment]
I think it should be "check pkg and work.bin file/zRIF key", it makes much more sense.

also, the "more" in multiple places makes it grammatically wrong, please remove them

File: lang/en-gb.xml [hunk 1]
```diff
@@ -408,7 +409,8 @@ Please download and install it.</no_font_exist>
 			<select_bin_rif>Select work.bin/rif</select_bin_rif>
 			<successed_install_license>Successfully installed license.</successed_install_license>
 			<delete_bin_rif>Delete the work.bin/rif file?</delete_bin_rif>
-			<check_bin_rif>Please check correct the work.bin/rif file.</check_bin_rif>
+			<failed_install_license>Failed to install license.
```
Comment (Zangetsu38) [inline lang/en-gb.xml:412]: no need "more" ^^

**Comment by Croden1999** [comment]
ready to review

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**